CMakeLists.txt

```diff
@@ -102,6 +102,11 @@ include_directories(.)
 include_directories(${CMAKE_BINARY_DIR})
 
```

Comment:
Typo in commit message: `unsable` should be.. unstable? unusable? :)

Also "!5616" does not get linkified, but I think "#5616" would.

Reply:
I used ! because the issue number was at the beginning of the line and was treated as a comment. Googled around and found `git commit --amend --cleanup=scissors` which avoids that problem, so that's neat :)

Typo deleted since you fixed the underlying issue that it was talking about anyway